### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by getssh**
I worked on the portfolio desk-top version
- Added media query for mid-width 768px
- I used a grid and flexbox for positioning and spacing 
- rem and percent have been used for a better responsiveness 
- contact form collector has been connected

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,102 +17,120 @@
...
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+      </section>
+      <ul class="social">
+          <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li><a href=""><i class="fa-regular fa-hand-peace"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href=""><i class="fa-solid fa-m"></i></a></li>
```
Comment (NduatiKagiri) [inline index.html:46]: - [ ] [ OPTIONAL ] listst1'

-   To make your interface look better, please consider removing this default style. You can achieve that by adding this CSS rule : `.social li { list-style: none; }`

File: index.html [hunk 2]
```diff
@@ -17,102 +17,120 @@
...
+        <img src="./images/ImgPlaceholder.png" alt="featured project" class="first-project">
+      </div>
+      <div class="pro-detail">
+        <h2 class="project-title">Multi-Post Stories</h2>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="used-tools">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
```
Comment (NduatiKagiri) [inline index.html:68]: - [ ] [ OPTIONAL ] Kindly apply the right Font to these list items as specified in the Figma design(Inter). Remember to make this change on all the cards in this section.

**Review by NduatiKagiri — APPROVED** [review]
Hi @getssh 👋 ,

# APPROVED 💯  

You have done a commendable job passing this project milestone on the first attempt👏👏. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

you did

## Highlights ✔️ 
✅ No Linters errors
✅ GitHub Flow observed
✅ Good PR description
✅ Good/Smooth UI

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by getssh** [comment]
Optional changes applied

**Pull request merged**